data/apizoo/nirkal.json

```diff
@@ -1,11 +1,30 @@
 [
   {
-    "user_name": "example_username_url",
-    "api_name": "Kubernetes API",
-    "url": "https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.27/api/openapi-spec/swagger.json",
+    "user_name": "nirkal", 
+    "api_name": "Kubernetes Pod List", 
+    "api_call": "io.k8s.api.core.v1.PodList()", 
+    "api_version": "1.0", 
```

Comment:
The api_version should correspond to the K8S API version right? In this case, 1.27.